Source/Core/Core/ConfigManager.cpp

```diff
@@ -608,6 +609,7 @@ void SConfig::LoadCoreSettings(IniFile& ini)
   core->Get("EnableCustomRTC", &bEnableCustomRTC, false);
   // Default to seconds between 1.1.1970 and 1.1.2000
   core->Get("CustomRTCValue", &m_customRTCValue, 946684800);
+  core->Get("DisableSignatureChecks", &m_disable_signature_checks, false);
```

Comment:
It would probably be slightly less confusing to have a `SignatureChecks` setting instead.